doc/specs/stdlib_array.md

```diff
@@ -0,0 +1,89 @@
+---
+title: array
+---
+
+# The `stdlib_array` module
+
+[TOC]
+
+## Introduction
+
+Module for index manipulation and array handling tasks.
+
+## Procedures and methods provided
+
+
+### `trueloc`
+
+#### Status
+
+Experimental
+
+#### Description
+
+Turn a logical mask into an index array by selecting all true values.
+
+#### Syntax
+
+`call [[trueloc(function)]] (array[, lbound])`
```

Comment:
Is it a function of a subroutine?
In the demo, it is a function.

Reply:
```suggestion
`loc = [[trueloc(function)]] (array[, lbound])`
```